addons/binding/org.openhab.binding.solarlog/ESH-INF/binding/binding.xml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<binding:binding id="solarlog"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xmlns:binding="http://eclipse.org/smarthome/schemas/binding/v1.0.0"
+        xsi:schemaLocation="http://eclipse.org/smarthome/schemas/binding/v1.0.0 http://eclipse.org/smarthome/schemas/binding-1.0.0.xsd">
+
+    <name>SolarLog Binding</name>
+    <description>This is the binding for SolarLog. See the README.md for more information about its use.</description>
```

Comment:
Please remove "See the README.md for more information about its use." as this file is not available to the end user. Rather add a sentence of what SolarLog is or what it is for.